projects/recipe-page/phase-3-bootstrap-prompt.md

```diff
@@ -158,3 +158,11 @@ Let's clean up the navbar a little bit. You can remove things like the search ba
 ### Extensions
 
 Find some other Bootstrap components you can add to your webpage!
+
+### Real life usage of frameworks
```

Comment:
@AmaanRS, 

- Please move this new section under the first "NOTE" on the page.
- Please also remove the spaces between each bulleted item, to maintain uniformity amongst bullets across the markdown.
- Please add a bullet about "being able to explain in an interview setting about how they were able to adapt a project"

Reply:
I made the changes.